testing/integration/leak/leak_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
Nit: if we rename this file to something like `main_test.go` (and the same in the `serverless` package), it'll be easier to identify where the `TestMain` is defined in any integration test package because of the consistency in naming.